source/agora/test/GossipProtocol.d

```diff
@@ -27,18 +29,19 @@ unittest
     network.start();
     assert(network.getDiscoveredNodes().length == NodeCount);
 
-    Hash h1 = sha512Of("Message No. 1");
```

Comment:
This should remove the need to `import std.digest.sha` (L24) but let's do that in another PR because I don't want the CI to run another time.